mesheryctl/pkg/utils/error.go

```diff
@@ -597,34 +590,6 @@ func ErrRequestResponse(err error) error {
 		[]string{"Check your network connection and the status of Meshery Server via `mesheryctl system status`."})
 }
 
-func ErrMarshalStructToCSV(err error) error {
```

Comment:
Please check if similar errors and error details already exist in meshkit. If they do, use those instead of moving them there.